observability/distr_tracing/distr-tracing-rn.adoc

```diff
@@ -12,9 +12,163 @@ You can use the {TempoName} xref:../otel/otel-forwarding-telemetry-data.adoc#ote
+
+[NOTE]
+====
+The {TempoName} 3.4 is based on the open source link:https://grafana.com/oss/tempo/[Grafana Tempo] 2.<y>.0.
+====
+
+[id="distr-tracing_3-4_tempo-release-notes_technology-preview-features_{context}"]
+==== Technology Preview features
+
+This update introduces the following Technology Preview feature:
+
+* ???.
+* ???.
+* ???.
+* ???.
+* ???.
+* ???.
+* ???.
+* ???.
+* ???.
+* ???.
+
+:FeatureName: Each of these features
+include::snippets/technology-preview.adoc[leveloffset=+1]
+
+[id="distr-tracing_3-4_tempo-release-notes_new-features-and-enhancements_{context}"]
+==== New features and enhancements
+
+This update introduces the following enhancements:
+
+* ???.
+* ???.
+* ???.
+* ???.
+* ???.
+* ???.
+* ???.
+* ???.
+* ???.
+* ???.
+
+[id="distr-tracing_3-4_tempo-release-notes_deprecated-functionality_{context}"]
+==== Deprecated functionality
+
+* In the {TempoName} 3.4, ???.
+* In the {TempoName} 3.4, ???.
+* In the {TempoName} 3.4, ???.
+
+[id="distr-tracing_3-4_tempo-release-notes_removal-notice_{context}"]
+==== Removal notice
+
+In the {TempoName} 3.4, the FEATURE has been removed. Bug fixes and support are provided only through the end of the 3.? lifecycle. As an alternative to the FEATURE for USE CASE, you can use the ALTERNATIVE instead.
+
+[id="distr-tracing_3-4_tempo-release-notes_bug-fixes_{context}"]
+==== Bug fixes
+
+This update introduces the following bug fixes:
+
+* Before this update, ... . With this update, this issue is fixed by ... . (link:https://issues.redhat.com/browse/TRACING-????[TRACING-????])
+* Before this update, ... . With this update, this issue is fixed by ... . (link:https://issues.redhat.com/browse/TRACING-????[TRACING-????])
+* Before this update, ... . With this update, this issue is fixed by ... . (link:https://issues.redhat.com/browse/TRACING-????[TRACING-????])
+* Before this update, ... . With this update, this issue is fixed by ... . (link:https://issues.redhat.com/browse/TRACING-????[TRACING-????])
+
+[id="distr-tracing_3-4_tempo-release-notes_known-issues_{context}"]
+==== Known issues
+
+There is currently a known issue:
+
+* Currently, the {TempoShortName} fails on the {ibm-z-title} (`s390x`) architecture. (link:https://issues.redhat.com/browse/TRACING-3545[TRACING-3545])
+* Currently, ???. (link:https://issues.redhat.com/browse/TRACING-????[TRACING-????])
+* Currently, ???. (link:https://issues.redhat.com/browse/TRACING-????[TRACING-????])
+* Currently, ???. (link:https://issues.redhat.com/browse/TRACING-????[TRACING-????])
+
+[id="distr-tracing_3-4_jaeger-release-notes_{context}"]
+=== {JaegerName}
+
+The {JaegerName} 3.4 is provided through the link:https://catalog.redhat.com/software/containers/rhosdt/jaeger-operator-bundle/613b50f3a052bb39f2c5f31e/history[{JaegerOperator} Operator <x>.<y>.<z>].
+
+[NOTE]
+====
+The {JaegerName} 3.4 is based on the open source link:https://www.jaegertracing.io/[Jaeger] release 1.<y>.0.
```

Comment:
@pavolloffay, I understand that we will be sticking with one Jaeger version going forward, but could you please confirm which version it is?

Reply:
Copied.